src/signin.ts

```diff
@@ -100,9 +101,17 @@ export function getSigninRequestHeaders(options: SigninOptions = {}): StringMap
   if ('accessToken' in options && options.accessToken) {
     return { [HEADER_X_ACROLINX_AUTH]: options.accessToken };
   } else if (isSsoSigninOption(options)) {
+    let username = options.username;
+    let password = options.genericToken;
+    if (!isUrlEncoded(username)) {
```

Comment:
Why this extra check is needed?

Reply:
So we do not encode stuff twice, because then we have encoded it too often and the value changes:
<img width="312" alt="Screenshot 2025-01-16 at 16 49 26" src="https://github.com/user-attachments/assets/a43c9cf7-05f8-422c-9724-4be6b70e83ae" />